subworkflows/local/amp.nf

```diff
@@ -65,8 +74,38 @@ workflow AMP {
 
         AMP_HMMER_HMMSEARCH ( ch_in_for_amp_hmmsearch )
         ch_versions = ch_versions.mix(AMP_HMMER_HMMSEARCH.out.versions)
+        GUNZIP2 ( AMP_HMMER_HMMSEARCH.out.output )
+        ch_hmmout = GUNZIP2.out.gunzip
+                        .map {
+                            meta_id, meta_hmm ->
+                            def meta_hmmsearch = [:]
+                            meta_hmmsearch['id'] = meta_id['id']
+                            [meta_hmmsearch, meta_hmm]
+                        }
+
+        ch_ampcombi_input = ch_ampcombi_input.mix(ch_hmmout)
     }
 
+    //AMPCOMBI
+    ch_ampcombi_input_new = ch_ampcombi_input
+        .groupTuple()
+        .join(ch_faa_for_ampcombi )
+        .multiMap{
+            input: [ it[0], it[1] ]
+            faa: it[2]
+        }
+
+    AMPCOMBI( ch_ampcombi_input_new.input, ch_ampcombi_input_new.faa , [])
```

Comment:
Can you leave a comment why the last channel is empty `[]` for future us (...probably me) why we don't use it?

Reply:
The last channel is the amp_database that can be provided by a user to run AMPcombi, rather than the default which is downloading DRAMP db using this flag `--amp_database`. Should we add it in the schema as an ext. params? or leave it as it is?

Reply:
We just added `--amp_database` instead to a external argument to make it easier for the user to supply a custom database folder instead of the automatic DRAMP download